great-idea/css/index.css

```diff
@@ -63,4 +63,260 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+/*desktop.jpg CSS code starts here*/
+.container{
+	width: 1000px;
+	margin: 0 auto;
+	display: flex;
+	flex-direction: column;
+}
+/*navigation */
+nav{
+	display: flex;
+	justify-content: baseline;
+}
+
+nav a{
+	width: 20%;
+	margin-top: 20px;
+	color: grey;
+	text-decoration: none;
+}
+
+.logo{
+	display: flex;
+	margin-top: 10px;
+}
+/*header */
+.header div h1{
+	font-size: 60px;
+	text-align: center;
+	margin: 80px 30px 0;
+	display: flex;
+	/* background-color: aqua; */
+	
+}
+
+.header img{
+	/* background-color: aquamarine; */
+	display: flex;
+	margin: 20px;
+}
+
+header{
+	width: 100%;
+	display: flex;
+	/* background-color: deepskyblue; */
+}
+
+button{
+	display: flex;
+	margin: 20px 80px; 
+	width: 130px;
+	padding: 5px 25px;
+	box-sizing: border-box;
+	background-color: white;
+	border-style: ridge;
+	/* background-color: hotpink; */
+}
+
+.col{
+	height: 400px;
+	width: 45%;
+	margin: 0 2%;
+	display: flex;
+	justify-content: center;
+	/* background-color: pink; */
+}
+
+.col:first-child{
+	margin-left: 3%;
+	flex-direction: column;
+}
+/*sections */
+section{
+	margin: 0;
+	padding: 10px 0;
+	/* background-color: darkslategray; */
+}
+section div{
+	padding: 10px;
+	max-width: 30%;
+	margin: 0;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-evenly;
+	/* background-color: gray; */
+}
+
+/* 1st section */
+.features{
+	border-top: 1px solid black;
+	max-width: 100%;
+	margin: 0;
+	display: flex;
+	justify-content:space-between;
+	/* background-color: rgb(26, 143, 143); */
+}
+
+.features div{	
+	display: flex;
+	flex-direction: column;
+	justify-content:space-between;
+	max-width: 100%;
+	margin: 10px 0;
+	text-align: left;
+	/* background-color: rgb(69, 53, 139); */
+}
+
+/* 2nd section */
+.services{
+	border-bottom: 1px solid black;
+	max-width: 100%;
+	margin: 0;
+	display: flex;
+	justify-content:space-between;
+	/* background-color: rgb(148, 7, 243); */
+}
+
+.services div{
```

Comment:
I  want to call out how you are consistently using the idea of CSS specificity to target elements within parent containers. This is exactly how you should be doing things - the less IDs and classes that we have floating around the better!